### ISSUE TITLE
the evaluation results on validation set is very low

### ISSUE THREAD

**Issue by beyoursel** [SmartForest-no/ForestFormer3D#20]
I have trained the ForestFormer3d on my machine with a 4090 GPU, batch_size=1, radius=8m, and other params is the same with the original code, but I found that the evaluation results on val set is very low. However , the result on test set is normal. Is there any problem?

<img width="733" height="219" alt="Image" src="https://github.com/user-attachments/assets/03e188a6-700e-469f-87a2-9e8b9abc04e0" />

**Comment by bxiang233** [comment]
> I have trained the ForestFormer3d on my machine with a 4090 GPU, batch_size=1, radius=8m, and other params is the same with the original code, but I found that the evaluation results on val set is very low. However , the result on test set is normal. Is there any problem?
> 
> <img alt="Image" width="733" height="219" src="https://private-user-images.githubusercontent.com/66890499/532794388-03e188a6-700e-469f-87a2-9e8b9abc04e0.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzQwMTc3NzAsIm5iZiI6MTc3NDAxNzQ3MCwicGF0aCI6Ii82Njg5MDQ5OS81MzI3OTQzODgtMDNlMTg4YTYtNzAwZS00NjlmLTg3YTItOWU4YjlhYmMwNGUwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjAzMjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwMzIwVDE0Mzc1MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZlZTY1MGVlOTIyOTA2N2FhMTA5NDQ0ZTE2ZmMwYTFmZTUxZThmNjc1NTE1MmZmNDhjNGUyOTAwYWYyOGZlMmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.jJMIXfzm0M3sm877J-IaW9rRRjywLLxbEmzW8QsKrr4">

Hi, I did not modify the original OneFormer3D code for computing validation metrics, so the validation results were actually incorrect and not really meaningful. Later, Atakan tried to fix this part, but I did not have time to rerun the experiments and verify the updated validation results myself.

During training, I mainly tracked the training loss. As long as the loss kept decreasing, I considered the training normal, and I stopped when it became stable. I always saved the last model, and I did not use a save-best strategy based on validation metrics.

So your model is already trained properly. Could you try testing your current trained model directly and also visualize the predictions? The results should be similar to the checkpoint I provided. If the test results are still poor, please feel free to post them here, and I can help analyze them.

Best,
Binbin
